### DISCUSSION TITLE
How to remove a GitHub sponsor with the "Pending" tag

### DISCUSSION THREAD

**Discussion by tembbo** [community/community#142891] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I accidentally (for some reason) signed up for a GitHub sponsor account, and now I can't delete it. Anyone who knows how?

**Comment by TzivyaFireman** [comment]
To delete your GitHub Sponsors account, you can follow these steps:

Sign in to your GitHub account.

Click on your profile picture in the upper right corner of the screen and select "Settings" from the dropdown menu.

Look for the "Sponsorship" or "Sponsorships" option in the sidebar of the settings. It might be under "Account" or "Billing."

If you have a sponsorship plan or joined any program, look for an option to cancel or delete your subscription.

**Reply by tembbo** [reply]
I don't see anything about sponsorships in settings, the only thing I saw was payment info which I removed.

**Comment by tembbo** [comment, marked as answer]
1. Go to https://support.github.com/contact/account
2. Select your github account and email.
3. Choose "GitHub Sponsors" in the dropdown
4. Enter "Fully Deleting GitHub Sponsors Account" as the subject
5. In the message box, write something in the lines of:

    > I recently set up a GitHub Sponsors account but would like to delete it completely, including the associated GitHub Sponsors Stripe 
    account. Previously, deleting a Sponsors account only resulted in a "Disabled" status rather 
    than full removal. Looking for a way to permanently delete it.

@Chitransh-1, hope this helps
